recipes/isce3/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "isce3" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isce-framework/isce3/archive/v{{ version }}.tar.gz
+  sha256: 9de6a6839818f2553e2f31e49ca5335a3c41b77e4482777e17f376d131da589b
+  patches:
+    - fix-find-numpy.patch
+    - fix-missing-includes.patch
+
+build:
+  number: 0
+  skip: True  # [win or py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - cython
+    - gtest
+    - ninja
+  host:
+    - cereal
+    - eigen
+    - fftw
+    - gdal
+    - h5py
+    - numpy {{ numpy }}
```

Comment:
If this is just using pybind11 for numpy support, then you might not need numpy in `host` at all.

If you're using the C API directly, see https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Regardless the `{{ numpy }}` can be removed.

Reply:
Numpy support is needed for some remaining cython components (which will eventually be phased out in favor of pybind11). So I think pin_compatible is the right mechanism, if I understand correctly